src/settings/UserListView.ts

```diff
@@ -24,70 +23,69 @@ import {isUpdateForTypeRef} from "../api/main/EventController"
 import {ButtonN, ButtonType} from "../gui/base/ButtonN"
 import {compareGroupInfos} from "../api/common/utils/GroupUtils"
 import {GENERATED_MAX_ID} from "../api/common/utils/EntityUtils"
-import {showNotAvailableForFreeDialog} from "../misc/SubscriptionDialogs"
 import {ListColumnWrapper} from "../gui/ListColumnWrapper"
 import {assertMainOrNode} from "../api/common/Env"
 import {locator} from "../api/main/MainLocator"
 import Stream from "mithril/stream";
+import {showNotAvailableForFreeDialog} from "../misc/SubscriptionDialogs"
```

Comment:
Since you are redoing the file, could you also do imports with `.js` please? Webstorm still ignores the preference.